doc/python/box-plots.md

```diff
@@ -71,6 +71,49 @@ fig = px.box(df, x="time", y="total_bill", points="all")
 fig.show()
 ```
 
+### Choosing The Algorithm For Computing Quartiles
+
+By default, quartiles for box plots are computed using a linear algorithm method (see #10 listed on [http://www.amstat.org/publications/jse/v14n3/langford.html](http://www.amstat.org/publications/jse/v14n3/langford.html) for more details). However, you can also choose to use an `exclusive` or an `inclusive` algorithm to compute quartiles. 
```

Comment:
For a reference, you could also link to the wikipedia page https://en.wikipedia.org/wiki/Quartile which describes all three algorithms and is more likely to stay in the future. But the amstat page is useful for other algorithms so you could have both references.

Reply:
Sure, added a link to the wiki page.